<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.8.0 -->
<title>DELTA lab</title>
<meta property="og:title" content="Home" />
<meta name="author" content="DELTA" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="DELTA lab at TONGJI University" />
<meta property="og:description" content="DELTA lab at TONGJI University" />
<meta property="og:site_name" content="DELTA" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="DELTA" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"WebPage","author":{"@type":"Person","name":"DELTA"},"description":"DELTA lab at TONGJI University","headline":"Home","url":"/home/"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/assets/css/style.css"><link type="application/atom+xml" rel="alternate" href="/feed.xml" title="DELTA lab" /></script>
<script>
  window['ga-disable-G-8125DSHDFE'] = window.doNotTrack === "1" || navigator.doNotTrack === "1" || navigator.doNotTrack === "yes" || navigator.msDoNotTrack === "1";
  window.dataLayer = window.dataLayer || [];
  function gtag(){window.dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-8125DSHDFE');
</script>


</head>
<body><header class="site-header">
        <div class="wrapper"><a class="site-title" rel="author" href="/">DELTA lab</a><nav class="site-nav">
          <input type="checkbox" id="nav-trigger" class="nav-trigger" />
          <label for="nav-trigger">
            <span class="menu-icon">
              <svg viewBox="0 0 18 15" width="18px" height="15px">
                <path d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.032C17.335,0,18,0.665,18,1.484L18,1.484z M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.032C17.335,6.031,18,6.696,18,7.516L18,7.516z M18,13.516C18,14.335,17.335,15,16.516,15H1.484 C0.665,15,0,14.335,0,13.516l0,0c0-0.82,0.665-1.483,1.484-1.483h15.032C17.335,12.031,18,12.695,18,13.516L18,13.516z"/>
              </svg>
            </span>
          </label>
  
          <div class="trigger"><a class="page-link" href="/">Home</a><a class="page-link" href="/publications/">Publications</a><a class="page-link" href="/contact/">Contact</a></div>
        </nav></div>

  </header>
  <div>
    <img src="/pics/tongji.jpg" alt="" class="nav-bar-img">
  
  </div>
  
  <main class="page-content" aria-label="Content" style="margin:5vh 1vw 1vh 5vw ">
      <div class="wrapper">
        <article class="post">

  
  <header class="post-header">
    <h1 class="post-title">Publications</h1>
  </header>
  

  <div class="post-content">
    <ul>
    <p style="padding-right: 10px; padding-left: 15px; text-decoration: underline;"> <strong><em>Scientific popular articles on Bioinformatics and Artificial Intelligence:</em></strong></p>  

    <p><strong>[1]</strong>. <span style="font-family:微软雅黑;">熊召平, <strong>刘琦</strong>, 郑明月, 蒋华良等, <a href="https://dl.ccf.org.cn/article/articleDetail.html?type=xhtx_thesis&_ack=1&id=6133602370078720" target="_self">联邦学习能否打破新药研发的反摩尔定律</a>, 中国计算机学会通讯</span>(CCCF), 2022.</p>
    <p><strong>[2]</strong>. <span style="font-family:微软雅黑;"><strong>刘琦#</strong>, <a href="https://mp.weixin.qq.com/s/Jo6meVR1VGamXJX0koGUqg">人工智能应用于新药研发的范式转变</a>, 知识分子公众号科普</span>, 2018.</p>
    <p><strong>[3]</strong>. <span style="font-family:微软雅黑;"><strong>刘琦#</strong>, <a href="https://mp.weixin.qq.com/s?__biz=MzIyNDA2NTI4Mg==&mid=2655422772&idx=1&sn=bd500ac05a447c1d94616beb537f5211&chksm=f3a68b59c4d1024ff059cb780b432bcd44bbc18b379e437a92320f6aed95af2f115a161c8147&scene=21#wechat_redirect">化学界诞生了一个“AlphaGO"</a>, 中国计算机学会通讯</span>(CCCF), 2016.</p>
    <p><strong>[4]</strong>. <span style="font-family:微软雅黑;"><strong>刘琦#</strong>, <a href="http://www.ccf.org.cn/c/2016-12-01/550967.shtml">生物信息学研究的思考</a>, 知识分子公众号科普</span>, 2018.</p>
    
    </ul>
    
    <ul>  
    <p style="padding-right: 10px; padding-left: 15px; text-decoration: underline;"> <strong><em>Representative papers in the area of Bioinformatics:</em></strong></p>  

      <p style="padding-right: 10px; padding-left: 15px;"> <strong>—— 2026 ——</strong></p>
    <p><strong>[1]</strong>. Keyan Ding et al, <strong>Qi Liu#</strong>, Haofen Wang#, Qiang Zhang# and Huajun Chen#, Bridging Data and Discovery: A Survey on Knowledge Graphs in AI for Science, <strong><em>National Science Review</em></strong>, Advance Access, 2026.</p>

      <p style="padding-right: 10px; padding-left: 15px;"> <strong>—— 2025 ——</strong></p>
    <p><strong>[1]</strong>. Zhiting Wei et al, <strong>Qi Liu#</strong>, Benchmarking algorithms for generalizable single-cell perturbation response prediction, <strong><em>Nature Methods</em></strong>, Advance Access, 2025.</p>
    <p><strong>[2]</strong>. Kejing Dong et al, <strong>Qi Liu#</strong>, Benchmarking multi-slice integration and downstream applications in spatial transcriptomics data analysis, <strong><em>Genome Biology</em></strong>, Advance Access, 2025.</p>
    <p><strong>[3]</strong>. Yuli Gao et al, <strong>Qi Liu#</strong>, Weakly-supervised peptide-TCR binding prediction facilitates neoantigen identification, <strong><em>Cell System</em></strong>, Advance Access. 2025.</p>
    <p><strong>[4]</strong>. Shaliu Fu et al, <strong>Qi Liu#</strong>, Benchmarking single-cell multi-modal data integrations, <strong><em>Nature Methods</em></strong>, Advance Access, 2025.</p>
    <p><strong>[5]</strong>. Yicheng Gao et al, <strong>Qi Liu#</strong>, Causal disentanglement for single-cell representations and controllable counterfactual generation, <strong><em>Nature Communications</em></strong>, Advance Access, 2025.</p>
    <p><strong>[6]</strong>. Xichen Liao, <strong>Qi Liu#</strong>, Guohui Chuai#, PrimeNet: rational design of Prime editing pegRNAs by deep learning, <strong><em>Briefings in Bioinformatics</em></strong>. Advance Access, 2025.</p>
    <p><strong>[7]</strong>. Dongdong Mao et al, <strong>Qi Liu#</strong> Gonghui Li#, Heterogeneous driving effect guides personalized tumor treatment targeting N6-methylatedadenine, <strong><em>Cancer Research</em></strong>, Advance Access, 2025.</p>
    <p><strong>[8]</strong>. Xiaojie Chen et al, <strong>Qi Liu#</strong>, SpaLinker identifies phenotype-associated spatial tumor microenvironment features by integrating bulk and spatial sequencing data, <strong><em>Cell Genomics</em></strong>, Advances Access, 2025.</p>
    <p><strong>[9]</strong>. Hao Xu et al, <strong>Qi Liu#</strong>, Jing-Dong Jackie Han#, Biomedical data and AI, <strong><em>Science China-Life Science</em></strong>. Advance Access, 2025.</p>
    <p><strong>[10]</strong>. Pengxiang Wang et al, <strong>Qi Liu#</strong>, Jia Fan#, Xinrong Yang#, Exploring Morphological Heterogeneity of Circulating Tumor Cells: Machine Learning-Based Approach for Cell Identification and Prognostic Implications, <strong><em>Science Bulletin</em></strong>, Advance Access, 2025.</p>

    <p style="padding-right: 10px; padding-left: 15px;"> <strong>—— 2024 ——</strong></p>
    <p><strong>[1]</strong>. Wenhui Li et al, <strong>Qi Liu#</strong>, Discovering CRISPR-Cas system with self-processing pre-crRNA capability by foundation models, <strong><em>Nature Communications</em></strong> 2024.</p>
    <p><strong>[2]</strong>. Zhiting Wei et al, <strong>Qi Liu#</strong>, PerturBase: a comprehensive database for single-cell perturbation data analysis and visualization, <strong><em>Nucleic Acids Research</em></strong> 2024.</p>
    <p><strong>[3]</strong>. Yicheng Gao, <strong>Qi Liu#</strong>, Delineating the cell types with transcriptional kinetics, <strong><em>Nature Computational Science</em></strong>. 2024.</p>
    <p><strong>[4]</strong>. Yicheng Gao et al, <strong>Qi Liu#</strong>, Toward subtask decomposition-based learning and benchmarking for genetic perturbation outcome prediction and beyond, <strong><em>Nature Computational Science</em></strong>, 2024.</p>
    <p><strong>[5]</strong>. <strong>Qi Liu*</strong>, Translating “DELTA” into precision therapy, <strong><em>Medicine Plus</em></strong>, 2024.</p>
    <p><strong>[6]</strong>. Liangzi Mengfang et al, <strong>Qi Liu#</strong>, Genomics-guided Representation Learning for Pathologic Pan-cancer Tumor Microenvironment Subtype Prediction, <strong><em>MICCAI</em></strong>. 2024.</p>
    <p><strong>[7]</strong>. Yicheng Gao et al., <strong>Qi Liu#</strong>, Unified cross-modality integration and analysis of T-cell receptors and T-cell transcriptomes by low-resource-aware representation learning, <strong><em>Cell Genomics</em></strong>. Advance Access, 2024.</p>
    <p><strong>[8]</strong>. Yunda Si et al, <strong>Qi Liu#</strong>, Luonan Chen#, Foundation models in Molecular Biology, <strong><em>Biophysics Reports</em></strong>, Advance Access, 2024.</p>
    <p><strong>[9]</strong>. Zhang Wei et al, <strong>Qi Liu#</strong>, Zhouxi Shi#, Qi Dai#, scHybridBERT: integrating gene regulation and cell graph for spatial-temporal dynamics in single-cell clustering. <strong><em>Briefings in Bioinformatics</em></strong>, Advance Access, 2024.</p>
    <p><strong>[10]</strong>. Fengying Sun et al., <strong>Qi Liu#</strong>… ShiTie Liu#, Single- Cell Omics: experimental workflow, data analyses and applications, <strong><em>Science China - Life Sciences</em></strong>. Advance Access, 2024.</p>
    <p><strong>[11]</strong>. Tianyi Ding et al, <strong>Qi Liu#</strong>, He Zhang #, Inter3D: Capture of TAD reorganization endows variant patterns of gene transcription, <strong><em>Genomics Proteomics & Bioinformatics</em></strong>, Advance Access, 2024.</p>
    <p><strong>[12]</strong>. Bin Duan et al., <strong>Qi Liu#</strong>, Multi-slice Spatial Transcriptome Domain Analysis with SpaDo, <strong><em>Genome Biology</em></strong>. Advance Access, 2024. </p>
      
    <p style="padding-right: 10px; padding-left: 15px;"> <strong>—— 2023 ——</strong></p>
    <p><strong>[1]</strong>. Chen Tang et al., <strong>Qi Liu#</strong>, Personalized tumor combination therapy optimization using the single-cell transcriptome, <strong><em>Genome Medicine</em></strong>. Advance Access, 2023.</p>
    <p><strong>[2]</strong>. Qichang Chen et al., <strong>Qi Liu#</strong>, Genome-wide CRISPR off-target prediction and optimization using RNA-DNA interaction fingerprints,<strong><em>Nature Communications</em></strong>. Advance Access, 2023.</p>
    <p><strong>[3]</strong>. Yicheng Gao et al., <strong>Qi Liu#</strong>, Reply to: The pitfalls of negative data bias for the T-cell epitope specificity challenge, <strong><em>Nature Machine Intelligence</em></strong>. Advance Access, 2023.</p>
    <p><strong>[4]</strong>. Xiaowen Wang et al, <strong>Qi Liu#</strong>, Qin Liu#, A complete graph-based approach with multi-task learning for predicting synergistic drug combinations, <strong><em>Bioinformatics</em></strong>, Advance Access, 2023. </p>
    <p><strong>[5]</strong>. Yicheng Gao et al., <strong>Qi Liu#</strong>, Pan-Peptide Meta Learning for T-Cell Receptor-Antigen Binding Recognition, <strong><em>Nature Machine Intelligence</em></strong>. Advance Access, 2023. </p>
    <p><strong>[6]</strong>. Qinhu Zhang et al., <strong>Qi Liu#</strong>,  Deshuang Huang#, Computational prediction and characterization of cell-type-specific and shared binding sites, <strong><em>Bioinformatics</em></strong>. Advance Access, 2023. </p>

    <p style="padding-right: 10px; padding-left: 15px;"> <strong>—— 2022 ——</strong></p>
    <p><strong>[1]</strong>.Gongchen Zhang et al., <strong>Qi Liu#</strong>, Systematic exploration of optimized base-editing gRNA design and pleiotropic effects with BExplorer, <strong><em>Genomics Proteomics & Bioinformatics</em></strong>. Advance Access, 2022. </p>
    <p><strong>[2]</strong>. Shaoqi Chen et al, <strong>Qi Liu#</strong>, Privacy-preserving integration of multiple institutional data for single-cell type identification with scPrivacy, <strong><em>Science China-Life Sciences</em></strong>, Advance Access, 2022. </p>
    <p><strong>[3]</strong>. Zhiting Wei et al., <strong>Qi Liu#</strong>, DrSim: Similarity learning for transcriptional phenotypic drug discovery, <strong><em>Genomics Proteomics & Bioinformatics</em></strong>. Advance Access, 2022. </p>
    <p><strong>[4]</strong>. Qinchang Chen et al, <strong>Qi Liu#</strong>, Toward a molecular mechanism-based prediction of CRISPR-Cas9 targeting effects, <strong><em>Science Bulletin</em></strong>, Advance Access, 2022. </p>
    <p><strong>[5]</strong>. Qinhu Zhang et al, <strong>Qi Liu#</strong>, Deshuang Huang#, Base-resulotion prediction of transcription  factor binding signals by a deep learning framework, <strong><em>Plos Computational Biology</em></strong>, Advance Access, 2022. </p>
    <p><strong>[6]</strong>. Dongyu Xue et al, <strong>Qi Liu#</strong>, X-MOL: large-scale pre-training for molecular understanding and diverse molecular analysis, <strong><em>Science Bulletin</em></strong>, Advance Access, 2022. </p>
    <p><strong>[7]</strong>.Gaoyang Li et al, <strong>Qi Liu#</strong>, A deep generative model for multi-view profiling of single cell RNA-seq and ATAC-seq data, <strong><em>Genome Biology</em></strong>, Advance Access, 2022. </p>

    <p style="padding-right: 10px; padding-left: 15px;"> <strong>—— 2021 ——</strong></p>
    <p><strong>[1]</strong>. Yukang Gong et al, <strong>Qi Liu#</strong>, DeepReac+: Deep active learning for quantitative modeling of organic chemical reactions, <strong><em>Chemical Science</em></strong>, Advance Access, 2021. </p>
    <p><strong>[2]</strong>. Xiaowen Wang, <strong>Qi Liu#</strong>, Qin Liu#, PRODeepSyn: integrating protein-protein interaction network with omics data to predict anticancer synergistic drug combinations, <strong><em>Briefings in Bioinformatics</em></strong>, Advance Access, 2021. </p>
    <p><strong>[3]</strong>. Tengbo Zhang et al, <strong>Qi Liu#</strong>, Ping Wang#, iCRISEE: an integrative  analysis of CRISPR screen by reducing false positive hits, <strong><em>Briefings in Bioinformatics</em></strong>, Advance Access, 2021. </p>
    <p><strong>[4]</strong>. Biyu Zhang et al, <strong>Qi Liu#</strong>, The tumor therapy landscape of synthetic lethality, <strong><em>Nature Communications</em></strong>, Advance Access, 2021. </p>
    <p><strong>[5]</strong>. Bin Duan et al, <strong>Qi Liu#</strong>, Integrating multiple references for single cell assignment, <strong><em>Nucleic Acids Research</em></strong>, Advance Access, 2021. </p>
    <p><strong>[6]</strong>. Qinhu Zhang et al, <strong>Qi Liu#</strong>, Deshuang Huang#, Locating transcription factor binding sites by Fully Convolutional Neural Network, <strong><em>Briefings in Bioinformatics</em></strong>, Advance Access, 2021. </p>
    <p><strong>[7]</strong> . Xiangyong Li et al, <strong>Qi Liu#</strong>, Hao Ye#, Benchmark HLA genetyping and clarifying HLA impact on survival in tumor immunotherapy,  <strong><em>Molecular Oncology</em></strong>, Advance Access, 2021. </p>
      
    <p style="padding-right: 10px; padding-left: 15px;"> <strong>—— 2020 ——</strong></p>
    <p><strong>[1]</strong>. Bin Duan et al, <strong>Qi Liu#</strong>, Learning for single cell assignment, <strong><em>Science Advances</em></strong>, Advance Access, 2020. (<strong><span style="color: rgb(0, 0, 0); font-size: 16px; background-color: rgb(0, 255, 0);">Selected as one of the top ten advances in the application of bioinformatics in China in 2020 </span></strong>) </p>
    <p><strong>[2]</strong>. Jifang Yan et al, <strong>Qi Liu#</strong>, Benchmarking and integrating CRISPR off-target detection and prediction, <strong><em>Nucleic Acids Research</em></strong>, Advance Access, 2020. </p>
    <p><strong>[3]</strong>. Shaoqi Chen et al, <strong>Qi Liu#</strong>, FL-QSAR: a federated learning based QSAR prototype for collaborative drug discovery, <strong><em>Bioinformatics</em></strong>, Advance Access, 2020. </p>
    <p><strong>[4]</strong>. Zhiting Wei et al, <strong>Qi Liu#</strong>, iDMer: an integrative and Mechanism-driven response system for identifying compound interventions for sudden virus break, <strong><em>Briefings in Bioinformatics</em></strong>, Advance Access, 2020. </p>

    <p style="padding-right: 10px; padding-left: 15px;"> <strong>—— 2019 ——</strong></p>
    <p><strong>[1]</strong>. Zhiting Wei et al, <strong>Qi Liu#</strong>, The landscape of tumor fusion neoantigens: a pan-cancer analysis, <strong><em>iScience</em></strong>, Advance Access, 2019. </p>    
    <p><strong>[2]</strong>. Chi Zhou et al, <strong>Qi Liu#</strong>, pTuneos: prioritizing Tumor neoantigens from next-generation sequencing data, <strong><em>Genome Medicine</em></strong>, Advance Access, 2019. </p>    
    <p><strong>[3]</strong>. Han Zhao et al, <strong>Qi Liu#</strong>, MetaMed: Linking microbiota functions with medicine therapeutics, <strong><em>mSystems</em></strong>, Advance Access, 2019. </p>  
    <p><strong>[4]</strong>. Chi Zhou et al, <strong>Qi Liu#</strong>, Towards in silico identification of tumor neoantigens in immunotherapy, <strong><em>Trends in Molecular Medicine</em></strong>, Advance Access, 2019. (<strong><span style="color: rgb(0, 0, 0); font-size: 16px; background-color: rgb(0, 255, 0);">Selected as one of the Best Review Article in Cell Trends 2019!</span></strong> <a href="http://crosstalk.cell.com/blog/best-reviews-we-published-in-2019-part-2">Report Link</a>) </p>    
    <p><strong>[5]</strong>. Yuli Gao et al, <strong>Qi Liu#</strong>, Data Imbalance in CRISPR off-target prediction, <strong><em>Briefings in Bioinformatics</em></strong>, Advance Access, 2019. </p>    
    <p><strong>[6]</strong>. Bin Duan et al, <strong>Qi Liu#</strong>, Model based Understanding of Single-cell CRISPR Screening, <strong><em>Nature Communications</em></strong>, Advance Access, 2019. (<strong><span style="color: rgb(0, 0, 0); font-size: 16px; background-color: rgb(0, 255, 0);">Selected as one of the top ten advances in the application of bioinformatics in China in 2019</span></strong>) </p>    
    <p><strong>[7]</strong>. Chenyu Zhu et al, <strong>Qi Liu#</strong>, C3: Consensus Cancer Driver Gene Caller, <strong><em>Genomics, Proteomics & Bioinformatics</em></strong>, Advance Access, 2019. </p>
      
    <p style="padding-right: 10px; padding-left: 15px;"> <strong>—— 2018 ——</strong></p>
    <p><strong>[1]</strong>. Dongyu Xue et al, <strong>Qi Liu#</strong>, Advances and challenges in deep generative models for de novo molecule generation, <strong><em>WIREs Computational Molecule Science</em></strong>, Advance Access, 2018. </p>
    <p><strong>[2]</strong>. Guohui Chuai et al, <strong>Qi Liu#</strong>, DeepCRISPR: optimized CRISPR guide RNA design by deep learning, <strong><em>Genome Biology</em></strong>, Advance Access, 2018. (<a href="https://f1000.com/prime/733509871"><span style="font-family: &quot;times new roman&quot;, serif; font-size: 16px; color: rgb(0, 0, 0); background-color: rgb(0, 255, 0);"> F1000 Recommendation</a></span>) </p>

    <p style="padding-right: 10px; padding-left: 15px;"> <strong>—— 2017 ——</strong></p>
    <p><strong>[1]</strong>. Ke Chen et al, <strong>Qi Liu#</strong>, Towards in-silico prediction of the immune-checkpoint blockade response, <strong><em>Trends in Pharmacological Sciences</em></strong>, Advance Access, 2017. <strong>(Most read article in the latest 30 days after publication!) </strong> </p>
    <p><strong>[2]</strong>. Jifang Yan et al, <strong>Qi Liu#</strong>, Benchmarking CRISPR on-target sgRNA design, <strong><em>Briefings in Bioinformatics</em></strong>, Advance Access, 2017. </p>
    <p><strong>[3]</strong>. Jifang Yan et al, <strong>Qi Liu#</strong>, Metatopics: an integration tool to analyze microbial community profile by topic model,  <strong><em>BMC Genomics</em></strong>, Advance Access, 2017. </p>

    <p style="padding-right: 10px; padding-left: 15px;"> <strong>—— 2016 ——</strong></p>
    <p><strong>[1]</strong>. Guohui Chuai, Jifang Yan et al, <strong>Qi Liu#</strong>, Deciphering relationship between microhomology and in-frame mutation occurence in human CRISPR-based gene knockout, <strong><em>Molecular Therapy-Nucleic Acids</em></strong>, Advance Access, 2016. <strong>(Featured Article ! ) </strong>  </p>
    <p><strong>[2]</strong>.  Jian Ma et al, <strong>Qi Liu#</strong>, Han Xu# and X. Shirley Liu#, CRISPR-DO: A genome-wide CRISPR designer and optimizer for multiple species, <strong><em>Bioinformatics</em></strong>, Advance Access, 2016. </p>
    <p><strong>[3]</strong>. Guo-hui Chuai, Qi-Long Wang, <strong>Qi Liu#</strong>, In-silico meets in-vivo: towards computational CRISPR-based sgRNA design, <strong><em>Trends in Biotechnology</em></strong>, Advance Access, 2016. <strong>(Most read article in the latest 30 days after publication!) </strong>  </p>

    <p style="padding-right: 10px; padding-left: 15px;"> <strong>—— Before 2015 ——</strong></p>
    <p><strong>[1]</strong>. Haiping Wang, Quanquan Gu,Jia Wei, Zhiwei Cao, <strong>Qi Liu#</strong>, Mining drug-disease relationships as a complement to medical genetics-based drug repositioning: Where a recommendation system meets GWAS, <strong><em>Clinical Pharmacology & Therapeutics</em></strong>, Advance Access, 2015. </p>
    <p><strong>[2]</strong>. Yi Sun, Zhen Sheng, Chao Ma, Kailin Tang, Ruixin Zhu, Zhuanbin Wu, Ruling Shen, Jun Feng, Dingfeng Wu, Danyi Huang, Dandan Huang, Jian Fei#, <strong>Qi Liu#</strong>, Zhiwei Cao#, Combining genomic and network characteristics for extended capability in predicting synergistic drugs for cancer, <strong><em>Nature Communications</em></strong>, Advance Access, 2015. </p>
    <p><strong>[3]</strong>. WZ, LJ, KT, HP W, RX Z, WJ, ZW, <strong>Qi Liu#</strong>, When drug discovery meets web searching: learning to rank for ligand-based virtual screening, <strong><em>Journal of Cheminformatics</em></strong>, Advance Access, 2015. </p>
    <p><strong>[4]</strong>. Haoqi Sun, Haiping Wang, Ruixin Zhu, Kailin Tang, Qin Gong, Juan Cui, Zhiwei Cao, <strong>Qi Liu#</strong>, iPEAP: integrating multiple omics and genetic data for pathway enrichment analysis, <strong><em>Bioinformatics</em></strong>, Advance Access online, 2013. </p>
    <p><strong>[5]</strong>. Yi Sun, Ruixin Zhu, Hao Ye, Jing Zhao, Yujia Chen, <strong>Qi Liu#</strong> and Zhiwei Cao#, Towards a bioinformatics analysis of anti-Alzheimer’s herbal medicines from a target network perspective, <strong><em>Briefings in Bioinformatics</em></strong>, May;14(3):327-43, 2012. </p>
    <p><strong>[6]</strong>. Qi Liu, Han Zhou, Ruixin Zhu, Ying Xu and Zhiwei Cao, Reconsideration of in-silico siRNA design from a perspective of heterogeneous data integration: problems and solutions, <strong><em>Briefings in Bioinformatics</em></strong>, Advance Access, 2012. </p>
    <p><strong>[7]</strong>. Hong Kang, Zhen Sheng, Ruixin Zhu, Qi Huang, <strong>Qi Liu#</strong> and Zhiwei Cao#, A virtual drug screen schema based on multi-view similarity integration and ranking aggregation, 26;52(3):834-43, <strong><em>J. Chem. Inf. Model</em></strong>. 2012. </p>
    <p><strong>[8]</strong>. Qi Liu, V. Olman, Huiqing Liu, Xiuzi Ye, Shilun Qiu, Ying Xu, RNACluster : an integrated tool for RNA secondary structure comparison and clustering, <strong><em>Journal of Computational Chemistry</em></strong>, 26(9), 1517-1526, 2008. </p>
    </ul>

    <ul>
    <p style="padding-right: 10px; padding-left: 15px; text-decoration: underline;"> <strong><em>Representative papers in the area of Machine Learning:</em></strong></p>  
      
    <p><strong>[1]</strong>. Xiaoxiao Shi, <strong>Qi Liu</strong>, Wei Fan, and Philip S. Yu, Transfer across Completely Different Feature Spaces via Spectral Embedding, <strong><em>IEEE Transactions on Knowledge and Data Engineering (TKDE)</em></strong>,  vol. 25, no. 4, pp. 906-918, 2011.</p>
    <p><strong>[2]</strong>. Xiaoxiao Shi, <strong>Qi Liu</strong>, Wei Fan, Philip Yu, and Ruixin Zhu, Transfer Learning on Heterogenous Feature Spaces via Spectral Transformation, <strong><em>IEEE International Conference on Data Mining (ICDM 2010)</em></strong>,1049-1054, 2010.</p>
    <p><strong>[3]</strong>. Xiao Xiao shi, <strong>Qi Liu</strong>, Wei Fan,Qiang Yang, Philip S.Yu. Predictive Modeling with heterogeneous Sources, <strong><em>the 2010 SIAM International Conference on Data Mining (SDM’2010)</em></strong>, 814-825, 2010.</p>
    </ul>

    <p>(#corresponding author. )</p>

  </div>

</article>
      </div>
    </main><footer class="site-footer h-card">
    <data class="u-url" href="/"></data>
  
     <div class="wrapper">
  
      <div class="footer">
          <ul class="contact-list">
            <li class="p-name">Qi Liu</li>
            <li><a class="u-email" href="mailto:qiliu@tongji.edu.cn">qiliu@tongji.edu.cn</a></li>
            <li><p>DELTA lab</p></li>
          </ul>
        
      </div>
  
    </div>
  
  </footer>
  </body>

</html>
